Source/Core/ArcGisImageServerTerrainProvider.js

```diff
@@ -200,11 +202,15 @@ define([
      * @param {Number} x The X coordinate of the tile for which to request geometry.
      * @param {Number} y The Y coordinate of the tile for which to request geometry.
      * @param {Number} level The level of the tile for which to request geometry.
+     * @param {Boolean} [throttleRequests=true] True if the number of simultaneous requests should be limited,
+     *                  or false if the request should be initiated regardless of the number of requests
+     *                  already in progress.
+     * @param {Number} [distance] The distance of the tile from the camera, used to prioritize requests.
      * @returns {Promise.<TerrainData>|undefined} A promise for the requested geometry.  If this method
      *          returns undefined instead of a promise, it is an indication that too many requests are already
      *          pending and the request will be retried later.
      */
-    ArcGisImageServerTerrainProvider.prototype.requestTileGeometry = function(x, y, level) {
+    ArcGisImageServerTerrainProvider.prototype.requestTileGeometry = function(x, y, level, throttleRequests, distance) {
```

Comment:
I still think this is a good first step:

> Or maybe to start, just replace both of these with a `requestPriority` that can be `undefined`.

But as I look at more of the code, the long-term approach could be that the `priority` is an object with a bunch of optional properties (`distance` being the most obvious), and the caller fills out as many properties as it knows.  Then the request scheduler knows how to sort based on the incomplete `priority` objects.  This would also allow things like "this request must happen right away (so perhaps we could central all requests even those that happen "immediately").